<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeasureMainWindow</class>
 <widget class="QMainWindow" name="MeasureMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="divide_lib_filename">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>划分lib名称</string>
    </property>
   </widget>
   <widget class="QPushButton" name="divide_dll">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>211</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>划分dll文件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="divide_lib">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>划分lib文件（以-gd区分）</string>
    </property>
   </widget>
   <widget class="QPushButton" name="divide_lib_dlib">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>划分lib文件（以d.lib区分)</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>100</y>
      <width>301</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="select_pc">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>选择点云文件</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
